src/main/java/Builder.java

```diff
@@ -41,7 +48,7 @@ public void build(String jsonPayload) {
 
         // maven install
         try {
-            String outputFilePath = "build_history/build_" + System.currentTimeMillis() + "_" + parser.getHeadCommitHash();
+            String outputFilePath = "build_history/" + parser.getHeadCommitHash() + " " + timestamp;
```

Comment:
Is the space between the hash and the timestamp intended?
Spaces in file names is preferably avoided, since it can cause issues. Could we replace with a underscore: '_'

Reply:
Yes it is, but we could change it to "_". We will need to change that in the History class too then.

Reply:
Yup! Just add a commit and we'll squash merge